spec/intergration/users_show_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'users#show', type: :feature do
+  describe 'Post' do
+    before(:all) do
+      @user = User.first!
+      @post1 = Post.first!
+    end
+
+    it "shows user's profile picture" do
+      visit(user_path(@user.id))
+      expect(page).to have_css('img[src*="https://dummyimage.com/600x400/000/fff"]')
+    end
+
+    it 'shows the users username' do
+      visit(user_path(@user.id))
+      expect(page).to have_content('frank')
+    end
+
```

Comment:
- [ ] Please add a test that asserts you can see the number of posts the user has written because you forgot to add it.

![image](https://user-images.githubusercontent.com/68796232/186075742-9624ff82-f50a-44c4-92ad-5ac874a425b4.png)